### DISCUSSION TITLE
Exporter Format State

### DISCUSSION THREAD

**Discussion by wadimsewo** [filamentphp/filament#13778] [Help] [answered]
### Package

Other

### Package Version

v3.2.97

### How can we help you?

Hello, first of all, thank you very much for the great work. FilamentPHP is a really cool tool that allows you to quickly build apps.

Regarding my question: I have an exporter and I’m trying to change a state, but I keep getting the following error: “An attempt was made to evaluate a closure for [Filament\Actions\Exports\ExportColumn], but [$value] was unresolvable”. Can someone help me with this?

`ExportColumn::make('from_date')
                ->label('Von Datum')
                ->formatStateUsing(function ($value) {
                    return $value->format('d.m.Y');
                }),`

`
 return ExportAction::make('export')
            ->label('Urlaubsanträge exportieren')
            ->modifyQueryUsing(function ($query) {
                return $query->where('year', $this->year);
            })
            ->modalHeading('Urlaubsanträge für das Jahr ' . $this->year . ' exportieren')
            ->formats([ExportFormat::Xlsx])
            ->requiresConfirmation()
            ->exporter(LeaveRequestExporter::class)
            ->color('gray')
            ->icon('heroicon-o-document-arrow-down');
`

**Comment by leandrocfe** [comment, marked as answer]
Use `string $state` instead of `$value`
